<script type="text/javascript" src="http://w.sharethis.com/button/buttons.js"></script>
<script type="text/javascript">stLight.options({publisher: "d135f460-3fc5-4802-8169-bd08e4734a09", doNotHash: false, doNotCopy: false, hashAddressBar: false});</script>
<span class='st_twitter_hcount' displayText='Tweet'></span>
<span class='st_facebook_hcount' displayText='Facebook'></span>
<span class='st_sina_hcount' displayText='Sina'></span>
<span class='st_linkedin_hcount' displayText='LinkedIn'></span>

## Diease or Drug analysis

+ Regulatory T Cells Orchestrate Similar Immune Evasion of Fetuses and Tumors in Mice, **The Journal of Immunology**, Arp. 2016, <http://dx.doi.org/10.4049/jimmunol.1501834>
+ In Silico Insight into Potential Anti-Alzheimer’s Disease Mechanisms of Icariin, **Int. J. Mol. Sci**, Jan. 2016 <http://dx.doi.org/10.3390/ijms17010113>
## Gene/Protein functional analysis

+ Single-Cell Co-expression Analysis Reveals Distinct Functional Modules, Co-regulation Mechanisms and Clinical Outcomes, **PLOS Computational Biology**, Apr. 2016 <http://dx.doi.org/10.1371/journal.pcbi.1004892>
+ Crosstalk of dynamic functional modules in lung development of rhesus macaques, **Mol. Biosyst.** Apr. 2016 <http://dx.doi.org/10.1039/C5MB00881F>
+ Comparative transcriptomics reveals the conserved building blocks involved in parallel evolution of diverse phenotypic traits in ants, **Genome Biology** Mar. 2016 <http://dx.doi.org/10.1186/s13059-016-0902-7>
+ FunTree: advances in a resource for exploring and contextualising protein function evolution, **Nucl. Acids Res.** Jan. 2016 <http://dx.doi.org/10.1093/nar/gkv1274>
+ Multi-omic measurement of mutually exclusive loss-of-function enriches for candidate synthetic lethal gene pairs, **BMC Genomics**, Jan. 2016 <http://dx.doi.org/10.1186/s12864-016-2375-1>

## Expression data processing

+ Robust Detection of Outlier Samples and Genes in Expression Datasets, **Journal of Proteomics & Bioinformatics** Feb. 2016 <http://dx.doi.org/10.4172/jpb.1000387>
+ Missing value imputation for microRNA expression data by using a GO-based similarity measure, **BMC Bioinformatics** Jan. 2016 <http://dx.doi.org/10.1186/s12859-015-0853-0>

## Protein-Protein Interaction

+ Computational prediction of virus–human protein–protein interactions using embedding kernelized heterogeneous data, **Mol. BioSyst**, Mar. 2016 <http://dx.doi.org/10.1039/C6MB00065G>
+ Investigating the impact human protein–protein interaction networks have on disease-gene analysis **International Journal of Machine Learning and Cybernetics** Feb. 2016 <http://dx.doi.org/10.1007/s13042-016-0503-5>
+ Computational probing protein–protein interactions targeting small molecules, **Bioinformatics** Jan. 2016 <10.1093/bioinformatics/btv528>
+ An integrative *C. elegans* protein–protein interaction network with reliability assessment based on a probabilistic graphical model, **Mol. Biosyst**, Jan. 2016 <http://dx.doi.org/10.1039/C5MB00417A>


